<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textDialog</class>
 <widget class="QDialog" name="textDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>&amp;Text</string>
    </attribute>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>233</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="fontSizeSpinner">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Text Input</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>481</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>100</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="textPreview">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>-1</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string>AbYz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxBold">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Bold</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxItalic">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Italic</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxStrike">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Strike out</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxUnderline">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Underline</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Colour</string>
     </property>
    </widget>
    <widget class="QLabel" name="textColour">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>50</width>
       <height>27</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(120,120,120)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAntialias">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>96</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Antialias</string>
     </property>
    </widget>
    <widget class="QLabel" name="emptyWarning">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Enter some text to continue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="alignleftButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>24</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../Icons.qrc">
       <normaloff>:/icons/assets/icons/aligntext_left.png</normaloff>:/icons/assets/icons/aligntext_left.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="aligncenterButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>24</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../Icons.qrc">
       <normaloff>:/icons/assets/icons/aligntext_center.png</normaloff>:/icons/assets/icons/aligntext_center.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="alignRightButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>24</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../Icons.qrc">
       <normaloff>:/icons/assets/icons/aligntext_right.png</normaloff>:/icons/assets/icons/aligntext_right.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBoxtextDialog">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
